src/Core/Components/DataGrid/FluentDataGridCell.razor.cs

```diff
@@ -61,4 +67,23 @@ protected override void OnInitialized()
 
     public void Dispose() => Owner.Unregister(this);
 
+    /// <summary />
+    internal async Task HandleOnCellClickAsync(string cellId)
```

Comment:
You could change that to `internal async Task HandleOnCellClickAsync(FluentDataGridCell<TGridItem> cell)` and calling this method via `@onclick="@(e => HandleOnCellClickAsync(this))"`

Reply:
That's actually pretty smart! Just one question: Is there a reason you didn't do this for OnRowClick as well? Or should I change that method, too, while I'm at it?

Reply:
The reason for this is probably that we've learned during this project :-)
Don't use this PR. It's better to have small PRs that add small features.